### PR TITLE
tests: add bug 1449 test

### PR DESCRIPTION
**Pull request by victorjulien**
Was private at the time but no need to keep it private.

https://redmine.openinfosecfoundation.org/issues/1449

### CODE REVIEW

File: tests/bug-1449-01/test.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+      count: 1
+      match:
+        event_type: alert
+        alert.signature_id: 2220017
+  - filter:
+      count: 1
+      match:
+        event_type: smtp
+        smtp.helo: bug.client
+        email.status: PARSE_ERROR
```
Comment (catenacyber) [inline tests/bug-1449-01/test.yaml:16]: Why do we want to enforce failure when we can succeed with the rust version ? (without overflowing)

Comment (victorjulien) [reply tests/bug-1449-01/test.yaml:16]: point here is to describe existing behavior. We won't update 6 or 7 parsers to rust

Comment (catenacyber) [reply tests/bug-1449-01/test.yaml:16]: So, should the check be done with lt-version: 8 ?

Comment (victorjulien) [reply tests/bug-1449-01/test.yaml:16]: I guess only when behavior for 8 changes

**Review by catenacyber — APPROVED** [review]
Ok for me.

The commit message could be improved (mention smtp and mime) instead of just the redmine id

**Comment by victorjulien** [comment]
Merged in #1495, thanks!

**Pull request closed without merging**